components/shared/IdentityPopover.vue

```diff
@@ -113,6 +115,19 @@ export default class IdentityPopover extends mixins(
   protected totalCollected = 0
   protected totalSold = 0
 
+  public async mounted() {
+    const { data } = await this.$apollo.query<{ events: Interaction[] }>({
+      query: buyEventByProfile,
+      client: 'subsquid',
```

Comment:
This query could not work on the bsx chain.

Reply:
Nice catch

Reply:
one doubt tho, why are we using subquery userStatsByAccount in rmrk and subsquid in bsx?

Reply:
> why are we using subquery userStatsByAccount in rmrk and subsquid in bsx?
not sure that you mean